doc/update0511.md

```diff
@@ -0,0 +1,57 @@
+### dPoW 0.5.11 update information
+
+- On your main node, delete the GLEEC coin and stop dPoW operations for it
+- On your main node, sync the CLC coin and start dPoW operations for it
+
+#### Stop the `GLEEC` chain on your main node
```

Comment:
Should be GleecBTC on 3P node (originally added in https://github.com/KomodoPlatform/dPoW/blob/master/doc/update032.md)

Reply:
thanks! will reopen a PR with correct instructions